node/deno_config.test.ts

```diff
@@ -0,0 +1,57 @@
+import { promises as fs } from 'fs'
+import { join } from 'path'
+
+import tmp from 'tmp-promise'
+import { expect, test } from 'vitest'
+
+import { getConfig } from './deno_config.js'
+
+test('Returns `undefined` if no config file is found', async () => {
+  const { cleanup, path } = await tmp.dir()
+  const config = await getConfig(path)
+
+  expect(config).toBeUndefined()
+
+  await cleanup()
+})
+
+test('Returns an empty object if the config file cannot be parsed', async () => {
+  const { path } = await tmp.dir()
+  const configPath = join(path, 'deno.json')
+
+  await fs.writeFile(configPath, '{')
+
+  const config = await getConfig(path)
+
+  expect(config).toEqual({})
+
+  await fs.rm(path, { recursive: true })
```

Comment:
according [to the docs](https://www.npmjs.com/package/tmp-promise), `tmp.dir()` also returns a `cleanup` function / a `withDir` helper that does the cleanup for us. That'd remove one line of "noise", and could make the tests easier to read.

Reply:
Done in https://github.com/netlify/edge-bundler/pull/256/commits/23bff755512d7dfbc3cb7d07727345ecb04db4dd.

Reply:
I wasn't using it because it needs the `unsafeCleanup` parameter when creating, which I didn't know about. I'll refactor the other tests in a follow-up.

Reply:
Oh, didn't see that in the docs either!